### PR TITLE
Create sensitivitylabel

### PR DESCRIPTION
**Pull request by SanjoyanM**
This is how this resource type will be used
https://graph.microsoft.com/v1.0/me/drive/items/ID?$select=**sensitivitylabel**.

### CODE REVIEW

**Comment by SanjoyanM** [comment]
> Thanks for your pull request SanjoyanM! I've added the "do not merge" label as a default status. Please remove this label and add the "content review" label to start a review. Once completed, label the reviewed PR with the "ready to merge" label when you are ready for the docs team to merge your pull request.
> 
> **Note:** Pull requests will not be merged until the "ready to merge" label is set.

Hi I dont have rights to change the "label" ... can you change the label for me please

File: api-reference/v1.0/resources/SensitivityLabel [hunk 1]
```diff
@@ -0,0 +1,46 @@
+---
+author: SanjoyanM
+ms.date: 05/05/2021
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:3]: Please refer to the guidance for metadata and make the metadata consistent. We don't use the ms.date attribute. The ms.prod value should be "sites-and-lists".

File: api-reference/v1.0/resources/SensitivityLabel [hunk 2]
```diff
@@ -0,0 +1,46 @@
+---
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:1]: Please make the file name all lowercase and add the .md file extension.

File: api-reference/v1.0/resources/SensitivityLabel [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+description: "The SensitivityLabel resource groups sensitivity-label related data items into a single structure."
+ms.prod: "sharepoint"
+doc_type: resourcePageType
+---
+
+# SensitivityLabel resource type
+
+Namespace: microsoft.graph
+
+The **SensitivityLabel** resource groups sensitivity-label related data items into a single structure.
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:15]: Resource topics generally begin with the verb without repeating the name of the resource in the description, so you can update to be "Group sensitivity label-related data items into a single structure." Note that the hyphen should be after label.

File: api-reference/v1.0/resources/SensitivityLabel [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+author: SanjoyanM
+ms.date: 05/05/2021
+title: SensitivityLabel resource type
+localization_priority: Normal
+description: "The SensitivityLabel resource groups sensitivity-label related data items into a single structure."
+ms.prod: "sharepoint"
+doc_type: resourcePageType
+---
+
+# SensitivityLabel resource type
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:11]: Resource names are lower camel case; please lowercase the S here and in the title and description fields.

File: api-reference/v1.0/resources/SensitivityLabel [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+```json
+{
+  "displayName": "string",
+  "id": "string",
+  "protectionEnabled": false
+}
+```
+
+## Properties
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:35]: We maintain a consistent order for the H2 headings; please move this section to come before the JSON representation section.

File: api-reference/v1.0/resources/SensitivityLabel [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+| Property          | Type                     | Description                                        
+|:------------------|:------------------------ |:-----------------------------------------------------------------------------------------
+| displayName       | string                   | Display name of the sensitivity label in format {parent-label name\\child-label name}.
+| id                | string                   | The unique identifier (guid) for sensitivity label.
+| protectionEnabled | boolean                  | Specifies if the label has an encryption policy attached or not.
+
+
+## Remarks
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:44]: Please remove this non-standard H2.

File: api-reference/v1.0/resources/SensitivityLabel [hunk 7]
```diff
@@ -0,0 +1,46 @@
...
+  "protectionEnabled": false
+}
+```
+
+## Properties
+
+| Property          | Type                     | Description                                        
+|:------------------|:------------------------ |:-----------------------------------------------------------------------------------------
+| displayName       | string                   | Display name of the sensitivity label in format {parent-label name\\child-label name}.
+| id                | string                   | The unique identifier (guid) for sensitivity label.
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:40]: Change guid to GUID (it's an acronym).

File: api-reference/v1.0/resources/SensitivityLabel [hunk 8]
```diff
@@ -0,0 +1,46 @@
...
+| protectionEnabled | boolean                  | Specifies if the label has an encryption policy attached or not.
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:41]: Boolean should be capitalized. Change the description to "Indicates whether the label..." for clarity/consistency.

**Review by Lauragra — DISMISSED** [review]
Please include a changelog entry with this PR and address editorial feedback.

File: api-reference/v1.0/resources/SensitivityLabel [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+{
+  "displayName": "string",
+  "id": "string",
+  "protectionEnabled": false
+}
+```
+
+## Properties
+
+| Property          | Type                     | Description
```
Comment (Lauragra) [inline api-reference/v1.0/resources/SensitivityLabel:37]: This entity is defined in the beta metadata but not in v1.0. Did you mean to add it to the /beta/resources folder? Also, in the metadata, the properties shown do not match the properties in this table. Please update the Properties table to list all properties of this resource.

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[f71a25a](https://github.com/SanjoyanM/microsoft-graph-docs/commits/f71a25a84b8f824c83fd950e7d4a39f3ceb360e2)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/sensitivitylabel?view=graph-rest-1.0&branch=pr-en-us-12714) (>= graph-rest-1.0) |

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109021610203412-12714%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=1sIecA6sB2JU8lOHWKLhUb0FGcO9sga7OviJmDxbPX4%3D&st=2021-09-02T16%3A07%3A36Z&se=2021-10-03T16%3A12%3A36Z&sp=r).

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[78bb39b](https://github.com/SanjoyanM/microsoft-graph-docs/commits/78bb39bd5f79f2598a42295c1cb3ceb7c1ab6bbd)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/sensitivitylabel?view=graph-rest-1.0&branch=pr-en-us-12714) (>= graph-rest-1.0) | [Details](#user-content-bcd5ed6e590d15439f2398f6295aa9a3-ec5b699c)

<a id="bcd5ed6e590d15439f2398f6295aa9a3-ec5b699c"></a>
### [api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md)
 - **Line 29, Column 4**: **[Warning-[hard-coded-locale](https://review.docs.microsoft.com/help/contribute/validation-ref/hard-coded-locale?branch=main)]** `````Link 'https://docs.microsoft.com/en-us/information-protection/develop/reference/mip-enums-and-structs#assignmentmethod-enum' contains locale code 'en-us'. For localizability, remove 'en-us' from links to most Microsoft sites.`````
 - **Line 44, Column 3**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../api/driveitem_setsensitivitylabel.md'.`````
 - **Line 2, Column 1**: **[Suggestion-[description-missing](https://review.docs.microsoft.com/help/contribute/validation-ref/description-missing?branch=main)]** `````Missing required attribute: 'description'.`````
 - **Line 29, Column 4**: **[Suggestion-[docs-link-absolute](https://review.docs.microsoft.com/help/contribute/validation-ref/docs-link-absolute?branch=main)]** `````Absolute link 'https://docs.microsoft.com/en-us/information-protection/develop/reference/mip-enums-and-structs#assignmentmethod-enum' will be broken in isolated environments. Replace with a relative link.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C2%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109021628047124-12714%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=iEwCMpN4MBm1DlI3URDMX8vtynS6eFrjASUY25i%2B5zE%3D&st=2021-09-02T16%3A25%3A38Z&se=2021-10-03T16%3A30%3A38Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[86675d3](https://github.com/SanjoyanM/microsoft-graph-docs/commits/86675d3484893fa8c11b300df42cf2f34d8389af)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/sensitivitylabel?view=graph-rest-1.0&branch=pr-en-us-12714) (>= graph-rest-1.0) | [Details](#user-content-bcd5ed6e590d15439f2398f6295aa9a3-3f0d29f3)

<a id="bcd5ed6e590d15439f2398f6295aa9a3-3f0d29f3"></a>
### [api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md)
 - **Line 47, Column 3**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../api/driveitem_setsensitivitylabel.md'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109070111328749-12714%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=8OetOCQYusisS9lzfiwZ82gjARUrGxMSd4HhtT3U864%3D&st=2021-09-07T01%3A08%3A13Z&se=2021-10-08T01%3A13%3A13Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

**Comment by jasonjoh** [comment]
Docs Build status updates of commit _[5b140cf](https://github.com/SanjoyanM/microsoft-graph-docs/commits/5b140cfefde8438a4709673286a720fd445d3e06)_: 

### :warning: Validation status: warnings


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md) | :warning:Warning | [View](https://review.docs.microsoft.com/en-us/graph/api/resources/sensitivitylabel?view=graph-rest-1.0&branch=pr-en-us-12714) (>= graph-rest-1.0) | [Details](#user-content-bcd5ed6e590d15439f2398f6295aa9a3-8c5b8b0c)

<a id="bcd5ed6e590d15439f2398f6295aa9a3-8c5b8b0c"></a>
### [api-reference/v1.0/resources/sensitivitylabel.md](https://github.com/SanjoyanM/microsoft-graph-docs/blob/patch-1/api-reference/v1.0/resources/sensitivitylabel.md)
 - **Line 47, Column 3**: **[Warning-[file-not-found](https://review.docs.microsoft.com/help/contribute/validation-ref/file-not-found?branch=main)]** `````Invalid file link: '../api/driveitem_setsensitivitylabel.md'.`````

For more details, please refer to the [build report](https://opbuilduserstoragepublic.blob.core.windows.net/report/2021%5C9%5C7%5C77e68b91-cf14-3558-43ca-0d6d735c492d%5CPullRequest%5C202109070119262950-12714%5Cworkflow_report.html?sv=2018-03-28&sr=b&sig=kt9wZRC%2BphGFMJbaF5IRK459vqH%2BW%2Frys3u9cFnqORM%3D&st=2021-09-07T01%3A16%3A04Z&se=2021-10-08T01%3A21%3A04Z&sp=r).

If you see build warnings/errors with permission issues, it might be due to single sign-on (SSO) enabled on Microsoft's GitHub organizations. Please follow instructions [here](https://review.docs.microsoft.com/en-us/help/onboard/sso-troubleshooting?branch=master) to re-authorize your GitHub account to Docs Build.

**Note:** Broken links written as relative paths are included in the above build report. For broken links written as absolute paths or external URLs, see the [broken link report](https://ops.microsoft.com/#/repos/77e68b91-cf14-3558-43ca-0d6d735c492d?tabName=brokenlinks).

**Note:** Your PR may contain errors or warnings unrelated to the files you changed. This happens when external dependencies like GitHub alias, Microsoft alias, cross repo links are updated. Please use these [instructions](https://review.docs.microsoft.com/en-us/help/onboard/faq?branch=master&tabs=pr-validation-error-solution#q-why-do-i-get-pr-validation-result-with-cryptic-error-or-warning-messages-not-related-to-the-changes-i-make-in-my-working-branch) to resolve them.

For any questions, please:<ul><li>Try searching the docs.microsoft.com <a href="https://review.docs.microsoft.com/en-us/help/?branch=master">contributor guides</a></li><li>Post your question in the <a href="https://teams.microsoft.com/l/channel/19%3a7ecffca1166a4a3986fed528cf0870ee%40thread.skype/General?groupId=de9ddba4-2574-4830-87ed-41668c07a1ca&amp;tenantId=72f98bf-86f1-41af-91ab-2d7cd011db47">Docs support channel</a></li></ul>

File: api-reference/v1.0/resources/filesensitivitylabel.md [hunk 10]
```diff
@@ -0,0 +1,60 @@
...
+
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+Group sensitivity label-related data items into a single structure for a given file.
+This facet can be used to fetch label id, protection status and sensitivity label name.
+
+It is available on the fileSensitivityLabel property of [driveItem][item-resource] resources.
+
```
[the remark this answers is not in the document]
Comment (chackman) [reply api-reference/v1.0/resources/filesensitivitylabel.md]: Is the "Methods" section required even if there are no methods defined on the type?

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 11]
```diff
@@ -0,0 +1,41 @@
+---
+author: SanjoyanM
+title: SensitivityLabel - sensitivity label of a file in SharePoint and OneDrive for Business
+description: This represents a sensitivity label of a file in SharePoint and OneDrive for Business.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:4]: Please include the rest of the headers: ms.prod, doc_type, ms.localizationpriority. Guidance here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103166/Metadata

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 12]
```diff
@@ -0,0 +1,41 @@
...
+---
+# SensitivityLabel type
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:6]: Line 6: Title should be: sensitivityLabel resource type.

Note the camel case and the name **resource type**.

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 13]
```diff
@@ -0,0 +1,41 @@
...
+
+The sensitivityLabel resource represents a sensitivity label of a file in SharePoint and OneDrive for Business.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:8]: Lines 4 and 8: These 2 descriptions should match. Please rephrase to use imperative voice:
* Represents a sensitivity label of...

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
...
+
+Here is a JSON representation of a sensitivityLabel resource.
+
+```json
+{
+  "id": "string",
+  "name": "string",
+  "assignmentMethod": [ "standard | privileged | auto"]
+}
+```
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:20]: This section should go at the end of the document.

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+
+## Properties
+
+| Property             | Type               | Description
+|:---------------------|:-------------------|:---------------------------------
+| id                   | string             | Id of the sensitivity label applied on the file. 
+| name                 | string             | Name of the sensitivity label applied on the file.
+| assignmentMethod     | AssignmentMethod | The assignment method of the label on the document. Whether the assignment of the label was done automatically, standard or as a privileged operation (The equivalent to an administrator operation).                                                                            |
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:28]: According to the metadata, this resource type has many more properties and 1 relationship:
<EntityType Name="sensitivityLabel" BaseType="graph.entity">
<Property Name="applicableTo" Type="graph.sensitivityLabelTarget"/>
<Property Name="applicationMode" Type="graph.applicationMode"/>
<Property Name="assignedPolicies" Type="Collection(graph.labelPolicy)"/>
<Property Name="autoLabeling" Type="graph.autoLabeling"/>
<Property Name="description" Type="Edm.String"/>
<Property Name="displayName" Type="Edm.String"/>
<Property Name="isDefault" Type="Edm.Boolean"/>
<Property Name="isEndpointProtectionEnabled" Type="Edm.Boolean"/>
<Property Name="labelActions" Type="Collection(graph.labelActionBase)"/>
<Property Name="name" Type="Edm.String"/>
<Property Name="priority" Type="Edm.Int32"/>
<Property Name="toolTip" Type="Edm.String"/>
<NavigationProperty Name="sublabels" Type="Collection(graph.sensitivityLabel)" ContainsTarget="true"/>

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+ For more information, see [assignmentmethod enum](/information-protection/develop/reference/mip-enums-and-structs#assignmentmethod-enum#assignmentmethod-enum).
+
+Represents the various methods to assign a sensitivity label. This value is stored in files and defines if the label can be overwritten or not.
+
+| Value               | Description
+|:--------------------|:------------------------------------------------------------------
+| standard            | Label assignment method is standard
+| privileged          | Label assignment method is privileged
+| auto                | Label assignment method is automatic
+| unknownFutureValue  | Marker value for future compatibility.
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:41]: Could you please clarify why are you including this section in the resource topic? It is not part of the structure for a resource topic. More details here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103174/Templates

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
+---
+author: SanjoyanM
+title: SensitivityLabel - sensitivity label of a file in SharePoint and OneDrive for Business
+description: This represents a sensitivity label of a file in SharePoint and OneDrive for Business.
+---
+# SensitivityLabel type
+
+The sensitivityLabel resource represents a sensitivity label of a file in SharePoint and OneDrive for Business.
+
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:9]: Please clarify the supported method for this resource. Does it support CRUD operations?

File: api-reference/v1.0/resources/sensitivitylabel.md [hunk 18]
```diff
@@ -0,0 +1,41 @@
+---
```
Comment (isvargasmsft) [inline api-reference/v1.0/resources/sensitivitylabel.md:1]: General comments:
Please include an entry for this resource type in the TOC.
Please include an entry in the Changelog.

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Thank you very much @SanjoyanM, 

There are some clarifications needed and files missing. Please let me know if you have any questions. I am here to help. 

Kind regards!

File: api-reference/v1.0/resources/sensitivitylabel.md (hunk 15 above)
Comment (SanjoyanM) [reply api-reference/v1.0/resources/sensitivitylabel.md:28]: Issac, not sure I understand what change is expected from documentation point here? Based on our last discussion can we resolve this or you are looking for specific information?

**Review by SanjoyanM — COMMENTED** [review]
@isvargasmsft Please review and let me know what is pending. Thanks in advance.

**Comment by isvargasmsft** [comment]
Moving this PR to https://github.com/microsoftgraph/microsoft-graph-docs/pull/14170

**Pull request closed without merging**